src/app/shared/doc-viewer/docs-api-module-import/docs-api-module-import.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, AfterViewInit, OnInit } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { CopierService } from "../../copier/copier.service";
```

Comment:
We should probably look at replacing the `CopierService` with the new https://material.angular.io/cdk/clipboard/overview.

Reply:
I was wondering about the `CopierService`, but left it that way for consistency. There are other components which are using it as well, for example the `ExampleViewer` component. It might be a good idea to switch to `Clipboard` with separate PR.